tests/unit/s2n_session_ticket_test.c

```diff
@@ -195,9 +195,11 @@ int main(int argc, char **argv)
         /* Set session state lifetime for 15 hours which is equal to the default lifetime of a ticket key */
         EXPECT_SUCCESS(s2n_config_set_session_state_lifetime(server_config, S2N_SESSION_STATE_CONFIGURABLE_LIFETIME_IN_SECS));
 
-        /* Add one ST key */
+        /* Add one session ticket key with an intro time in the past so that the key is immediately valid */
         POSIX_GUARD(server_config->wall_clock(server_config->sys_clock_ctx, &now));
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), now / ONE_SEC_IN_NANOS));
+        uint64_t key_intro_time = (now / ONE_SEC_IN_NANOS) - 1;
```

Comment:
From the failures we've seen, is one second sufficient? I know Wesley had some (sign overflowed :P) results he logged from one failure.

I'd also probably name this constant. Something like "ONE_SEC_DELAY" or "STEK_INTRO_DELAY".

Reply:
He wasn't able to recreate this particular failure. The replicable failure is another test. So I'm not sure how we'd know if this is sufficient other than "wait and see". 
But I can give it a name.